mk/config.mk

```diff
@@ -271,6 +271,9 @@ endif
 # Enable support for dynamically loaded user TAs
 CFG_WITH_USER_TA ?= y
 
+# Build user TAs included in this git
```

Comment:
i suggest to replace git with source tree

Reply:
Isn' t (This flag allows us to enable or disable building "User TAs")  is a bit confusing ?

Reply:
what do you mean?

Reply:
Applied, Thank you